### PR TITLE
preparing changelog for next beta

### PR DESCRIPTION
**Pull request by shiftkey**
There's a few small fixes ready to go out, and I figure getting the words ready before would be :gem:

### CODE REVIEW

**Comment by shiftkey** [comment]
@desktop/core if I can get a :thumbsup: for the updated changelog I'll merge this in and publish to beta.

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.3.5-beta1": [
+      "[Fixed] Disable delete button while deleting a branch - #5331",
+      "[Fixed] History now avoids calling log.showSignature if set in config - #5466",
+      "[Fixed] Revert workaround for tooltip issue on Windows, closes #3362 - #3362. Thanks @divayprakash!",
```
Comment (billygriffin) [inline changelog.json]: Is this a duplicate of the one two below it?

**Comment by shiftkey** [comment]
:apple:

**Comment by shiftkey** [comment]
VSTS is having [some uptime issues](https://twitter.com/tfsbuck/status/1036959100444311552) and I've not been able to trigger a new macOS build there, but we can live without that currently.

**Pull request merged**